travis-ci/vmtest/run_selftests.sh

```diff
@@ -28,6 +28,12 @@ test_verifier() {
 	travis_fold end test_verifier
 }
 
+bpftool_checks() {
```

Comment:
well, actually, bpftool_checks is not really invoked from anywhere...

Reply:
Silly me :cry:. Addressed in https://github.com/kernel-patches/vmtest/pull/23, and I'll update the libbpf/libbpf version right now.